src/common/utils/fhir/formatter.ts

```diff
@@ -0,0 +1,48 @@
+import { ProductDto } from 'src/common/dtos/product.dto';
+import { SystemProductDto } from 'src/common/dtos/system-product.dto';
+import { R4 } from '@ahryman40k/ts-fhir-types';
+
+export const getSystems = (products: ProductDto[]): Array<any> => {
```

Comment:
Using an Array of any defeats the purpose of the type system. Looks like an Array of strings could work.

Reply:
will change that